### ISSUE TITLE
Nullness Checker issues unnecessary warnings about LazyNonNull fields

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#107]
```
See class PptRelation1 in test case
checker-framework/checkers/tests/nullness/LazyInitialization.java
It contains a @SuppressWarnings annotation for now, but it shouldn't issue 
warnings even without the @SuppressWarnings annotation.
```

Original issue reported on code.google.com by `michael.ernst@gmail.com` on 25 Aug 2011 at 11:18
